baybe/recommenders/pure/bayesian/sequential_greedy.py

```diff
@@ -22,7 +22,7 @@
 
 @define
 class SequentialGreedyRecommender(BayesianRecommender):
-    """Recommender using sequential Greedy optimization.
+    """PureRecommender using sequential Greedy optimization.
```

Comment:
Since we do not refer to the class here, I would write "Pure recommender"

Reply:
changed